tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1785,18 +1785,19 @@ public function copyFilesAndFoldersRequest(string $user, string $fullUrl, array
 	 * @param string $fileContent
 	 * @param string $destination
 	 *
-	 * @return void
+	 * @return string[]
 	 * @throws GuzzleException
 	 */
 	public function userHasUploadedFile(
 		string $user,
 		string $spaceName,
 		string $fileContent,
 		string $destination
-	): void {
+	): array {
```

Comment:
filed id comes as array?

Reply:
yes it does

Reply:
@PrajwolAmatya can't we save the file id from the response set?

Reply:
> @PrajwolAmatya can't we save the file id from the response set?

any beneficial reason for doing that @SagarGi ?

Reply:
what if we do not need file id for other scenarios where we upload file?

Reply:
> @PrajwolAmatya can't we save the file id from the response set?

We are using the step `And we save it into "FILEID"` from `BehatVariablesContext` which we cannot modify. If we want to save the file id from the response set and use it in the `When` step then we might need to create a new step for it.